examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>Are we sure this is a valid assumption? Could you maybe create a plot with the tool computing TID for [0,1,2,4,8,16] days or so just to check that it is actually linear? (We can provide that plot here and to the reviewer to support the linearization)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Yes, it is not a valid assumption in general. However, I was reading that to calculate the protons and electron fluxes, we use the AE-8 ad AP-8 models that assume a constant radiation flux equal to the worst case. This is ok-ish for long mission (&gt; 6 months), but results are still very inaccurate when you go to few days. Check here: https://indico.cern.ch/event/635099/contributions/2570659/attachments/1456183/2249867/2_Radiation_Effects_and_RHA_ESA_Internal_Course_9_May_space_environment_HE.pdf</p><p> </p><p>Notice, that still you cannot simulate more than 30 days using SPENVIS (so 30 days is the best I could do). I can simulate for few days, so we align to SPENVIS but not necessarily to reality. </p>